src/booleans.js

```diff
@@ -1,62 +1,68 @@
 function negate(a) {
-  // your code here
+  return !a;
+  //
 };
 
 function both(a, b) {
-  // your code here
+  return a && b;
 };
 
 function either(a, b) {
-  // your code here
+  return a || b;
 };
 
 function none(a, b) {
-  // your code here
+  return !a && !b;
 };
 
 function one(a, b) {
-  // your code here
+  return (a&&!b) || (b&&!a);
 };
 
 function truthiness(a) {
-  // your code here
+  return !!a;
 };
 
 function isEqual(a, b) {
-  // your code here
+  return !!(a==b) ;
 };
 
 function isGreaterThan(a, b) {
-  // your code here
+  return (a>b);
 };
 
 function isLessThanOrEqualTo(a, b) {
-  // your code here
+  return (a<=b);
 };
 
 function isOdd(a) {
-  // your code here
+  return !((a%2)==0);
 };
 
 function isEven(a) {
-  // your code here
+  return ((a%2)==0);
 };
 
 function isSquare(a) {
-  // your code here
+  return a>=0&&Math.sqrt(a)%1===0;
 };
 
 function startsWith(char, string) {
-  // your code here
+  return string.startsWith(char);
 };
 
 function containsVowels(string) {
-  // your code here
-};
+  string.toLowerCase();
```

Comment:
The toLowerCase() method does not change the original string. So this line isn't actually doing anything. If you want to save the result of the lower case string you need to save the result of calling this method into a variable. For example `const lowerCaseString = string.toLowerCase()`. Javascript will run the function, but it won't save the result anywhere as it is currently written. Hope that makes sense?